tools/serverpod_cli/lib/src/config/config.dart

```diff
@@ -33,8 +33,15 @@ class GeneratorConfig {
     required List<String> relativeDartClientPackagePathParts,
     required this.modules,
     required this.extraClasses,
+    required this.serializeEnumValuesAsStrings,
   }) : _relativeDartClientPackagePathParts = relativeDartClientPackagePathParts;
 
+  /// The current [GeneratorConfig] instance.
+  static GeneratorConfig? _instance;
+
+  /// Get the current [GeneratorConfig] instance.
+  static GeneratorConfig? get instance => _instance;
```

Comment:
`static GeneratorConfing get instance {
  assert(_instance != null, 'GeneratorConfig is not loaded.');
  ...
}

Would be better.